assets/src/application/ui/layout/descriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { DescriptionListProps } from './types';
+import './styles.scss';
+
+const DescriptionList: React.FC<DescriptionListProps> = ({ dataSource }) => {
+	return (
+		<dl className='ee-description-list-grid'>
+			{dataSource.map(({ description, term }) => {
+				return (
+					<div>
+						<dt>{term}</dt>
+						<dd>{description}</dd>
+					</div>
```

Comment:
I still think this, combined with some of what you added in `EntityListLegend`, would be good to extract into separate components

ie:

```js
interface DescriptionListItemProps {
	className: string;
	description: string;
	term: string | EspressoIcon | ColorSwatch; // or React.Node or whatever
}

const DescriptionListItem: React.FC<DescriptionListItemProps > = ({ className, description, term }) => (
	<div className={`${className} ee-description-list-item`}>
		<dt>{term}</dt>
		<dd>{description}</dd>
	</div>
);
```

then the interface for `DescriptionList` could type for an array of `DescriptionListItem[]`

and `EntityListLegend` would do something like:

```js
	const iconsSource = icons.map(({ icon, description }) => {
		return (
			<DescriptionListItem
				className='ee-legend-description'
				description={description}
				term={
					<EspressoIcon aria-label={description} icon={icon as Icon} svgSize={18} />
				}
			/>
		);
	});
```

because then everything is components

Reply:
done here: https://github.com/eventespresso/event-espresso-core/pull/2561/commits/4e25febdd9d3cb85ab05fcd7f2dfe7665449afa0

Reply:
it is a bit tricky to setup:
`dataSource: DescriptionListItemProps[]`, have set for now:
`dataSource: JSX.Element[]`
***
more info in this thread: https://eventespresso.slack.com/archives/C080DDVLM/p1584627543023800